templates/ts/action/.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "debug.onTaskErrors": "abort",
```

Comment:
Since action is a generic name, is there any better name for this scenario/folder? e.g. message-extension-action?